_posts/en/newsletters/2025-12-19-newsletter.md

```diff
@@ -0,0 +1,1210 @@
+name: 2025-12-19-newsletter
+slug: 2025-12-19-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+
+excerpt: >
+  The eighth annual Bitcoin Optech Year-in-Review special summarizes notable
+  developments in Bitcoin during all of 2025.
+---
+
+{{page.excerpt}} It's the sequel to our summaries from [2018][yirs 2018],
+[2019][yirs 2019], [2020][yirs 2020], [2021][yirs 2021], [2022][yirs 2022],
+[2023][yirs 2023], and [2024][yirs 2024].
+
+## Contents
+
+* January
+  * [Updated ChillDKG draft](#chilldkg)
+  * [Offchain DLCs](#offchaindlcs)
+  * [Compact block reconstructions](#compactblockstats)
+* February
+  * [Erlay update](#erlay)
+  * [LN ephemeral anchor scripts](#lneas)
+  * [Probabilistic payments](#probpayments)
+* March
+  * [Bitcoin Forking Guide](#forkingguide)
+  * [Private block template marketplace to prevent centralizing MEV](#templatemarketplace)
+  * [LN upfront and hold fees using burnable outputs](#lnupfrontfees)
+* April
+  * [SwiftSync speedup for initial block download](#swiftsync)
+  * [DahLIAS interactive aggregate signatures](#dahlias)
+* May
+  * [Cluster mempool](#clustermempool)
+  * [Increasing or removing Bitcoin Core’s OP_RETURN size limit](#opreturn)
+* June
+  * [Calculating the selfish mining danger threshold](#selfishmining)
+  * [Fingerprinting nodes using addr messages](#fingerprinting)
+  * [Garbled locks](#garbledlocks)
+* July
+  * [Chain code delegation](#ccdelegation)
+* August
+  * [Utreexo draft BIPs](#utreexo)
+  * [Lowering the minimum relay feerate](#minfeerate)
+  * [Peer block template sharing](#templatesharing)
+  * [Differential fuzzing of Bitcoin and LN implementations](#fuzzing)
+* September
+  * [Details about the design of Simplicity](#simplicity)
+  * [Partitioning and eclipse attacks using BGP interception](#eclipseattacks)
+* October
+  * [Theoretical limitations on embedding data in the UTXO set](#arbdata)
+  * [Channel jamming mitigation simulation results and updates](#channeljamming)
+* November
+  * [Comparing performance of ECDSA signature validation in OpenSSL vs. libsecp256k1](#secpperformance)
+  * [Multiple discussions about restricting data](#restrictingdata)
+  * [Modeling stale rates by propagation delay and mining centralization](#stalerates)
+  * [BIP3 and the BIP process](#bip3)
+  * [Bitcoin Kernel C API introduced](#kernelapi)
+* December
+* Featured summaries
+  * [Vulnerabilities](#vulns)
+  * [Quantum](#quantum)
+  * [Soft fork proposals](#softforks)
+  * [Stratum V2](#stratumv2)
+  * [Major releases of popular infrastructure projects](#releases)
+  * [Bitcoin Optech](#optech)
+
+---
+
+## January
+
+{:#chilldkg}
+- **Updated ChillDKG draft:** Tim Ruffing and Jonas Nick updated their work on
+  a distributed key generation protocol (DKG) for use with the [FROST][news
+  frost bip] [threshold signature][topic threshold signature] scheme. ChillDKG
+  aims to provide similar recoverability features to existing descriptor
+  wallets.
```

Comment:
@reardencode: I think the FROST link isn't super relevant, suggest linking the bitcoinops article instead?

Commit to cherry-pick: https://github.com/stickies-v/bitcoinops.github.io/commit/f9c5bf99fbb3914096d74efc8a926838a77b50bc